app/assets/javascripts/redactor-rails/config.js

```diff
@@ -0,0 +1,22 @@
+window.init_redactor = function(){
+  var csrf_token = $('meta[name=csrf-token]').attr('content');
+  var csrf_param = $('meta[name=csrf-param]').attr('content');
+  var params;
+  if (csrf_param !== undefined && csrf_token !== undefined) {
```

Comment:
Identifier 'csrf_param' is not in camel case.<br>Identifier 'csrf_token' is not in camel case.